capstone/src/main/java/com/google/sps/servlets/HardcodedClubs.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps.servlets;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+
+/* Creates hardcoded club information for prototype. */
+public class HardcodedClubs {
```

Comment:
Nice job on this. This works, but an even simpler (and more stylistic) approach is to simply have this be a final class, and just define the map as a constant. These are often called "util" classes, and are used to hold static constants or methods that you can access from any class. Also, I would make the club name keys constants so you can access them easily from the servlet without hardcoding it in the servlet as well. So something like:

public final class PrototypeClubs {

private PrototypeClubs() {} // empty, private constructor because you don't need it for util classes

public static final String CLUB_1 = "Club 1";
public static final String CLUB_2  = ....
....

public static final ImmutableMap\<String, Club\> PROTOTYPE_CLUBS_MAP = 
ImmutableMap.of(CLUB_1, new Club(CLUB_1, ....), CLUB_2, new Club(CLUB_2, ...), .......);
}

Then you can just access the map in your servlet directly (after importing this class) by calling PROTOTYPE_CLUBS_MAP.get(CLUB_1). This way, you don't have to construct this new class every time you want to access the map.